src/engine/server/botnames.cpp

```diff
@@ -0,0 +1,127 @@
+#include "server.h"
+
+// C++11-style initializer
+std::vector<const char *> CServer::m_botnames = {
```

Comment:
Why the hell are you using std::vector for this?! What's against a fixed-size c-array?? The elements will always be the same anyway :o

PS: if you just do it because of the size() method: You can get the size of any c-array with `sizeof(array)/sizeof(array[0])`